filesystem/uimage.ksy

```diff
@@ -0,0 +1,103 @@
+meta:
+  id: u_image
+  endian: be
```

Comment:
A submission must include an open source license (`meta/license`) to be accepted. Please consider using CC0-1.0 to maximize possible application surface.